### PR TITLE
Add more integration tests

### PR DESCRIPTION
**Pull request by jasonLaster**
Adds several more steps to the integration tests:
+ enabling / disabling breakpoints
+ selecting a breakpoint (for source navigational purposes)
+ selecting a scope
+ step in, out, over ...

As I went, I refactored the debugger commands so that common selectors
were re-used to avoid painful test refactoring when the UI changes.

Writing the tests were actually pretty fun. Surprising/Unsuprisingly,
there were no hiccups along the way.

I cleaned up some of the CSS as I went so that the class names were
clearer.

### CODE REVIEW

File: public/js/actions/tests/loadSourceText.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ describe("loadSourceText", () => {
     Task.spawn(function* () {
       yield loadBadSource(store);
       const fooSourceText = getSourceText(store.getState(), "foo1");
-      expect(fooSourceText.get("error")).to.equal("poop");
+      expect(fooSourceText.get("error")).to.equal("failed to load");
```
Comment (jasonLaster) [inline public/js/actions/tests/loadSourceText.js]: got rid of the bad joke

File: public/js/components/Accordion.js [hunk 2]
```diff
@@ -37,7 +37,8 @@ const Accordion = React.createClass({
 
   renderContainer: function(item, i) {
     const { opened, created } = this.state;
-    const containerClassName = item.header.toLowerCase().replace(/\s/g, "-");
+    const containerClassName =
+          item.header.toLowerCase().replace(/\s/g, "-") + "-pane";
```
Comment (jasonLaster) [inline public/js/components/Accordion.js]: the `-pane` disambiguates the CSS from other class names

File: public/js/test/utils/debuggee.js [hunk 3]
```diff
@@ -106,7 +106,7 @@ function Debuggee() {
   };
 }
 
-const debuggeeStatement = `window.Debuggee = (${Debuggee})()`;
+const debuggeeStatement = `window.dbg = (${Debuggee})()`;
```
Comment (jasonLaster) [inline public/js/test/utils/debuggee.js]: changed `Debuggee` to `dbg` as it's a little easier to type

**Comment by jasonLaster** [comment]
@jlongster r? 

mostly cleanup from yesterday as I was doing a pass through the tests

**Comment by jasonLaster** [comment]
I'm noticing some issues with cypress on circle. It looks fairly consistent recently. The error logs indicate that it is probably Firefox starting that is causing the issue. I'll investigate a bit more next week.

**Comment by jlongster** [comment]
Looks fine to me. We probably should hold off on writing a lot more tests until I implement immutable tcomb types and type the app state. I feel like that may impact how we are doing things like hyrdating the store, but maybe not.

Merging as you are traveling today and since you asked for review I'm assuming it's ready

**Pull request merged**